platform-core-module/src/main/java/com/softicar/platform/core/module/CoreI18n.java

```diff
@@ -70,6 +70,8 @@ public interface CoreI18n extends EmfI18n {
 		.de("Am");
 	I18n0 ATTACHMENTS = new I18n0("Attachments")//
 		.de("Anhänge");
+	I18n0 AUTO_KILL = new I18n0("Auto Kill")//
```

Comment:
we could rename this "Automatic Abort" to get a better Germen translation like "Automatischer Abbruch"